docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -13,8 +13,9 @@ The following are its source, sink, and sanitizer definitions:
 
 *Source*
 
-1. @AuraEnabled methods
-2. Public methods on Visualforce Controllers
+1. `@AuraEnabled`-annotated methods
+2. Any method returning a `PageReference` object
```

Comment:
Though this was our first approach, didn't we change it after finding out that any public method on a VF controller can be accessed from the page?

Reply:
Nevermind, I just noticed the third point.